service/application/src/main/resources/application.yml

```diff
@@ -39,6 +39,27 @@ camel:
   springboot:
     use-mdc-logging: true
 
+kafka:
+  client:
+    config:
+      connections:
+        timeout:
+          value: 30_000
```

Comment:
Can you increase this to 60_000 please. I talked to Scott about this and he agrees.

Reply:
@gregmeldrum Sure thing!